### ISSUE TITLE
Asyncio 

### ISSUE THREAD

**Issue by ewi2345** [emendir/IPFS-Toolkit-Python#6]
Is there a reason why u have not used asyncio? Do u intend to do that in the near future?
IMHO, ur solution would significantly benefit from that. 
Best E.

**Comment by emendir** [comment]
I agree, but I'm afraid I have no plans for implementing asyncio at the moment because I have a lot of projects to prioritise amongst...

Feel free to contribute it.
Start with the ipfs_api module, as that's simplest, and ignore everything that's deprecated.
And write automated tests to ensure the behaviour is as expected.

`ipfs_datatransmission` and `ipfs_peers` will require a little more thought to apply asyncio to.
